### PR TITLE
add state for mental health changes

### PR DESCRIPTION
**Pull request by petermason**
# Description

WIP

## On what platform have you tested the change?

- [x] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevent
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: src/features/dashboard/DashboardLogVaccine.tsx [hunk 1]
```diff
@@ -33,17 +33,17 @@ export const DashboardLogVaccine: React.FC<Props> = ({ screenName }) => {
   };
 
   return (
-    <TouchableOpacity style={{ marginLeft: -14, marginTop: -30 }} onPress={onShareImage}>
```
Comment (selvach) [inline src/features/dashboard/DashboardLogVaccine.tsx:36]: Did you mean to check this in?

Comment (petermason) [reply src/features/dashboard/DashboardLogVaccine.tsx:36]: this was in develop? but looks like things have changed. spoke to Ben about this this morning!

Comment (petermason) [reply src/features/dashboard/DashboardLogVaccine.tsx:36]: er, no


> this was in develop? but looks like things have changed. spoke to Ben about this this morning!

er, no

**Review by bionara — APPROVED** [review]
nice one! great foundation

**Pull request merged**